bumpversion/cli.py

```diff
@@ -66,6 +67,42 @@
 ]
 
 
+def main(original_args=None):
+    args, known_args, root_parser, positionals = _parse_arguments_phase_1(original_args)
+    _setup_logging(known_args.list, known_args.verbose)
+    vcs_info = _determine_vcs_usability()
+    defaults = _determine_current_version(vcs_info)
+    explicit_config = None
+    if hasattr(known_args, "config_file"):
+        explicit_config = known_args.config_file
+    config_file = _determine_config_file(explicit_config)
+    config, config_file_exists, config_newlines, part_configs, files = _load_configuration(
+        config_file, explicit_config, defaults,
+    )
+    known_args, parser2, remaining_argv = _parse_arguments_phase_2(
+        args, known_args, defaults, root_parser
+    )
+    vc = _setup_versionconfig(known_args, part_configs)
+    current_version = _parse_current_version(known_args.current_version, vc)
+    context = _assemble_context(vcs_info)
```

Comment:
Given the function is now pretty much just a single line, should we inline it instead?

Reply:
Agreed